proposals-demo/CMakeLists.txt

```diff
@@ -0,0 +1 @@
+add_subdirectory(objectification)
```

Comment:
what's all this for?

Reply:
I wanted to improve some ease of use, but I accidentally pushed it to github, and then it was automatically synchronized to the PR stream. How can I remove the commit from 491cad9 in the RP stream?